static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -166,6 +166,28 @@ const registerIframeListener = (iframeId: string) => {
     });
 };
 
+const addServerSideRenderingTestParameterToLink = (rawUrl: string): string => {
+    const serverSideRenderingField = 'ssr';
+    const randomNumber = Math.random();
+    if (
+        randomNumber === undefined ||
+        config.get('switches.contributionsServerSideRenderingSwitch') === false
```

Comment:
You can pass a second param to config.get to set the default value:

```js
config.get('switches.contributionsServerSideRenderingSwitch', false)
```

This ensures that the function will always return a boolean. 😄

Reply:
nifty!